src/components/Movies.js

```diff
@@ -0,0 +1,51 @@
+import { Component } from "react";
+
+import './Movies.css'
+import React from "react";
+
+
+class Movies extends Component{
+    constructor(){
+        super()
+        this.state={
+            search :[],
+            film: null,
+        }
```

Comment:
```suggestion
        this.state={
            search :[],
            films: [],
        }
```
`this.state.films` (plural!!!) ends up being an array of **multiple** movie objects. Its name in state should reflect that it's NOT a single film, but an array of **several films**.